web/src/components/vis/VisPca.vue

```diff
@@ -67,9 +70,14 @@ export default {
       const {
         width,
         height,
-        points,
+        rawPoints,
       } = this.$props;
 
+      const points = rawPoints.x.map(p => ({
```

Comment:
@waxlamp is `x` a list of points in 5 (or other specified) dimensions?  Why bother computing more than 2 or 3?  Also, why name that list `x`?

Reply:
I think it's named `x` here because that's what `sklearn` calls it when the PCA is performed (but @jbeezley can confirm that).

As for why 5, that's the default setting for the `pca` endpoint, and I was too lazy to change it to 2 in my call. It should possibly be 2 by default but that depends on the typical use case for PCA (which may not be simply visualizing the first two PCs).